source/Calamari.Common/Calamari.Common.csproj

```diff
@@ -0,0 +1,61 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
It's hard to evaluate how much of this is required and if merged now would be missed in a future PR. I would either cut this down to the minimum required or not create this project at all yet.